haystack/components/generators/hugging_face_local.py

```diff
@@ -89,6 +89,9 @@ def __init__(
         huggingface_pipeline_kwargs = huggingface_pipeline_kwargs or {}
         generation_kwargs = generation_kwargs or {}
 
+        self.token = token
+        token = token.resolve_value() if token else None
```

Comment:
where is `token` used?

Reply:
In line 98, it gets added to the `huggingface_pipeline_kwargs` dict if it doesn't already exist.

Reply:
Ah the line was collapsed in the diff, thanks!